### ISSUE TITLE
[Kura 2.1.0 QA] Kura 2.1.0 installation dependencies

### ISSUE THREAD

**Issue by markoer** [eclipse/kura#850]
https://wiki.eclipse.org/Kura/Raspberry_Pi doesn't list ethtool and unzip dependencies on Debian/Raspberry Pi (both are mentioned in #316).

**Comment by MMaiero** [comment]
@markoer If you follow the instructions available [here](http://eclipse.github.io/kura/doc/raspberry-pi-quick-start.html) you'll see that the dependencies are completely resolved when performing `sudo apt-get install -f`.
However, we are aware that we should simplify our sites and provide a single source of information to our users and we are working in this direction.

**Comment by markoer** [comment]
In this case, let me rephrase.
Step 5 (Install Kura with:) [here](http://eclipse.github.io/kura/doc/raspberry-pi-quick-start.html) should probably mention that the dpkg command might fail and that that is perfectly normal installation flow.

**Comment by ctron** [comment]
A failing dpkg command should not be perfectly normal.

It is possible to install a dpkg with dependencies. You can e.g. use "gdebi" to install a DEB file with dependencies at the same time.

**Comment by cdealti** [comment]
Any confirm gdebi works?

**Comment by cdealti** [comment]
@markoer can you test the ``gdebi`` alternative?

**Comment by ctron** [comment]
So if you don't have `gdebi` installed you will need to run:

```
apt-get install gdebi-core
```

And then you can do:

```
gdebi my.deb
```

**Comment by markoer** [comment]
Gdebi seems to work nicely.

**Comment by cdealti** [comment]
Ok thanks. Can you take note of this in your validation spreadsheet and close this?
